Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -134,6 +135,8 @@ public void Initialize(TelemetryConfiguration configuration)
                                 this.SetComponentCorrelationHttpHeaders,
                                 this.ExcludeComponentCorrelationHttpHeadersOnDomains, 
                                 null);
+
+                            this.sqlClientDiagnosticSourceListener = new SqlClientDiagnosticSourceListener(configuration);
```

Comment:
if this is present in NET45/46 etc, then this code should be moved to InitializeForRuntimeInstrumentationOrFramework() method. Specifically - this should be enabled only if Profiler is determined to be NOT installed.